library/Backends/llvm/src/Juvix/Backends/LLVM/Pass/Types.hs

```diff
@@ -1,6 +1,6 @@
 module Juvix.Backends.LLVM.Pass.Types where
 
-import Juvix.Backends.LLVM.Primitive as Prim
+import qualified Juvix.Backends.LLVM.Primitive as Prim
```

Comment:
Nice shocked this compiled, as some of them were annoying with some type signatures